src/rcheevos/consoleinfo.c

```diff
@@ -604,12 +607,20 @@ static const rc_memory_regions_t rc_memory_regions_pc8800 = { _rc_memory_regions
 /* ===== PC Engine ===== */
 /* http://www.archaicpixels.com/Memory_Map */
 static const rc_memory_region_t _rc_memory_regions_pc_engine[] = {
+    { 0x000000U, 0x001FFFU, 0x1F0000U, RC_MEMORY_TYPE_SYSTEM_RAM, "System RAM" },
+    { 0x012000U, 0x041FFFU, 0x0D0000U, RC_MEMORY_TYPE_SYSTEM_RAM, "Super System Card RAM" }
+};
+static const rc_memory_regions_t rc_memory_regions_pc_engine = { _rc_memory_regions_pc_engine, 4 };
```

Comment:
There's only two entries not, so the `4` needs to be changed to `2`.

Additionally, removing the CD RAM block in the middle causes the RA addresses of the "Super System Card RAM" to change. The 0x012000U would have to be changed to `0x002000U`, and any games dependent on those addresses would have to have their code updated (which would break older versions of RetroArch that haven't received this change).

I thought the Super System Card was a CD-ROM expansion. In that case, it could be removed too.
> This was followed by the Super System Card released on October 26, an upgrade for the existing CD-ROM² add-on that serves as a replacement to the original System Card. 

If not, we'll probably need to put in a padding region to ensure the RA addresses are unchanged.